index.html

```diff
@@ -177,7 +177,7 @@ <h3 style="font-weight: bold">NewPipe - the smart streaming solution</h3>
                                     </svg>
                                     <span class="circle-text"><i class="fa fa-signal"></i></span>
                                 </div>
-                                <p class="text-center">Little Data Consumption</p>
+                                <p class="text-center">Less Data Consumption</p>
```

Comment:
Less than...? Not sure about this change. @TobiGr what's your opinion?

Reply:
less then normal app, since you can have sound only. Thats what I thought.

What about little otherwise? Litte in comparison to what? :D

Reply:
Well, less in comparison to what? I think "little" has a less "comparative" attitude than "less".

Please consider we have a large user base which doesn't use any Google apps.

I know it's a bit tedious to discuss about such details. I would've suggested low or reduced, but all those are "comparative" as well...

Reply:
I'd rather keep little

Reply:
Okay. I don't like Little for a reason I cant put my finger on, but I can definitely see the issue with a comparison. Will revert that.